docs/src/Submakefile

```diff
@@ -168,8 +195,24 @@ DOC_SRCS_EN := \
 	Master_Integrator.adoc \
 	Master_Developer.adoc
 
-LANGUAGES := $(shell sed -e's/#.*//' < $(DOC_DIR)/po4a.cfg | grep '^\[po4a_langs\]' | cut -d" " -f2-)
-LANGUAGES_MATCH := $(shell echo $(LANGUAGES) | tr " " "|")
+# Map two-letter codes to human-readable language names
+# (this can later be extended to handle zh_CN, zh_TW, etc.)
```

Comment:
You say that it can be expanded for zh_CN then include it? (and also state "two-letter")

Also the Chinese docs use all lower case zh_cn.

(I don't know enough to know if either matter)